microprofile/cors/src/main/java/io/helidon/microprofile/cors/CORSSupportMP.java

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright (c) 2020 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package io.helidon.microprofile.cors;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.function.Supplier;
+
+import javax.ws.rs.container.ContainerRequestContext;
+import javax.ws.rs.container.ContainerResponseContext;
+import javax.ws.rs.core.MultivaluedHashMap;
+import javax.ws.rs.core.MultivaluedMap;
+import javax.ws.rs.core.Response;
+
+import io.helidon.webserver.cors.CORSSupport;
+import io.helidon.webserver.cors.CrossOriginConfig;
+
+/**
+ * MP implementation of {@link CORSSupport}.
+ */
+class CORSSupportMP extends CORSSupport {
```

Comment:
Please use camel case: `CorsSupportMp` and `CorsSupport`

Reply:
Please update all classes in the implementation that use such a naming convention.